src/services/aggregateDonations/aggregateDonations.service.js

```diff
@@ -40,7 +40,7 @@ module.exports = function aggregateDonations() {
           count: { $sum: 1 },
           donations: { $push: '$_id' },
         })
-        .match({ totalAmount: { $gt: 0 } })
+        .match({})
```

Comment:
We cannot remove this line? it has no condition

Reply:
@aminlatifi  done